src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleEntries = sourceString.split(';');
```

Comment:
Consider using a more descriptive variable name for `styleEntries`. Since it represents the individual style declarations, a name like `styleDeclarations` would improve readability.